src/test/runTest.ts

```diff
@@ -0,0 +1,49 @@
+import * as path from 'path';
+
+import { runTests } from 'vscode-test';
+import * as fs from 'fs-extra';
+
+async function main() {
+
+	try {
+		// Loading config of testing workspace
+		const pythonPath = process.env.INMANTA_PYTHON_PATH;
+		const compilerVenv = process.env.INMANTA_COMPILER_VENV;
+		if (pythonPath === undefined) {
+			throw new Error("INMANTA_PYTHON_PATH has to be set");
+		}
+		if (compilerVenv === undefined) {
+			throw new Error("INMANTA_COMPILER_VENV has to be set");
+		}
```

Comment:
Why make this configurable?

Reply:
I thought that it might be useful for the test on venv, but given that i am not testing it here I should probably remove it

Reply:
I think that would be best, the person who picks up the ticket can then decide if it's required.

Reply:
Removed it in https://github.com/inmanta/vscode-inmanta/pull/33/commits/861a5702ef2483206fe5e16bfd83e9a51b630d47